omnibus/package-scripts/publish_image.sh

```diff
@@ -17,3 +18,8 @@ docker tag stackstate/${IMAGE_REPO}:${IMAGE_TAG} stackstate/${IMAGE_REPO}:latest
 
 docker login -u $DOCKER_USER -p $DOCKER_PASS
 docker push stackstate/${IMAGE_REPO}:${IMAGE_TAG}
+
+if ["$PUSH_LATEST= true" ]; then
+    echo 'Pushing release to latest'
+    docker push stackstate/${IMAGE_REPO}:latest
```

Comment:
`docker tag stackstate/${IMAGE_REPO}:${IMAGE_TAG} stackstate/${IMAGE_REPO}:latest`
can be moved inside this `if`

what you think?

Reply:
moved the tagging also to If block